### ISSUE TITLE
Fails to give an error if parser is not found

### ISSUE THREAD

**Issue by bebraw** [eslint/eslint#3209]
In short if parser is missing, it doesn't let you know that it's missing. [Example](https://github.com/bebraw/eslint-parser-fail)

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by gyandeeps** [comment]
This is what I got:
```sh
$ npm run lint

> eslint-parser-fail@1.0.0 lint c:\Users\gs025879\Documents\webstrom\eslint-parser-fail
> eslint . --ext .js --ext .jsx


index.js
  0:0  error  Cannot find module 'babel-eslint'

? 1 problem (1 error, 0 warnings)
```

**Comment by bebraw** [comment]
Ok, got it... There was a global install of `babel-parser` hanging around. I suppose it's safe to close this. :+1:
